src/presentation/react/core/components/metadata-table/MetadataTable.tsx

```diff
@@ -251,6 +264,10 @@ const MetadataTable: React.FC<MetadataTableProps> = ({
         updateFilters({ disableFilterRows });
     };
 
+    const changeOrgUnitsSyncAll = (value: boolean) => {
+        notifyModelSyncAllChange(value);
+    };
```

Comment:
Unless you have some specific idea in mind, you can just use notifyModelSyncAllChange, as there is no transformation here.

Reply:
I will leave a comment but it would be for a future feature where every metadataType can be "sync all" activated